perses/rjmc/geometry.py

```diff
@@ -17,7 +21,7 @@
 import logging
 logging.basicConfig(level = logging.NOTSET)
 _logger = logging.getLogger("geometry")
-_logger.setLevel(logging.INFO)
+_logger.setLevel(logging.WARNING)
```

Comment:
again... this should be INFO. if too much stuff is being printed, then too many logger comments are info when they shouldn't be